pkg/controller/apiserver/handlers/handler_test.go

```diff
@@ -68,61 +68,79 @@ var responses = []endpoint.Policies{
 				Name:      "policy2",
 			},
 		},
+		Egress: []antreatypes.NetworkPolicy{},
+		Ingress: []antreatypes.NetworkPolicy{},
 	},
 }
 
-func TestIncompleteOrInvalidArguments(t *testing.T) {
+// TestIncompleteArguments tests how the handler function responds when the user passes in a query command
+// with incomplete arguments (for now, missing pod or namespace)
+func TestIncompleteArguments(t *testing.T) {
 	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 	// sample selector arguments (right now, only supports podname and namespace)
-	podName, namespace := "podName", "namespace"
+	pod, namespace := "pod", "namespace"
 	// outline test cases with expected behavior
 	testCases := map[string]TestCase{
-		"Responds with empty list given no name and no namespace": {
-			handlerArgs:           []string{"", ""},
-			expectedStatus:  http.StatusOK,
-			expectedContent: responses[0],
+		"Responds with error given no name and no namespace": {
+			handlerRequest:           "",
+			expectedStatus:  http.StatusBadRequest,
```

Comment:
I am a bit confused because I see you using `http.StatusBadRequest` in the test, but I don't see you returning `StatusBadRequest` anywhere in the handler function. Am I missing something?

Reply:
This is correct, I am currently implementing this functionality in the handler. So, this test is not yet resolved, though this is the expected behavior I am looking for.